tests/test_labels.py

```diff
@@ -0,0 +1,14 @@
+import csv
+import os
+
+
+def test_labels():
+    with open('..\labels.csv') as csvfile:
```

Comment:
There's a function in utils to read labels, why do you keep rewriting the reading manually? :smile:

Reply:
I forget about the functions :P